firefox/manifest.json

```diff
@@ -20,5 +19,11 @@
       "js": ["firefox/src/install_notifier.js"],
       "run_at": "document_idle"
     }
-  ]
+  ],
+  "browser_specific_settings": {
+    "gecko": {
+      "id": "{b369ea25-4d25-4ecb-81dd-b09da203176a}",
```

Comment:
From where did you get this id? 🤔

Reply:
In Firefox, you navigate to:
about:debugging#/runtime/this-firefox

![Screenshot 2024-09-23 at 14 36 39](https://github.com/user-attachments/assets/ed2a0552-c594-49ba-b636-cd8cd4268270)